package.json

```diff
@@ -13,9 +13,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-solid-svg-icons": "^5.15.2",
+    "@fortawesome/vue-fontawesome": "^2.0.2",
```

Comment:
Waar worden deze gebruikt? Ik zie geen referenties.

Reply:
Waren al uit package.json verdwenen ? Nu in elk  geval wel.